{
  if (result.getForeignExceptionMessage() != null) {
    ForeignExceptionMessage exception=result.getForeignExceptionMessage().getForeignExchangeMessage();
    String msg=exception != null ? result.getExceptionFullMessage() : "no exception found";
    assertFalse(msg,result.isFailed());
  }
}
